testing/cffi0/test_verify.py

```diff
@@ -126,22 +137,14 @@ def test_Wconversion_double2int():
     _Wconversion("int sin(double);",
                  "#include <math.h>", libraries=lib_m)
 
-def test_rounding_1():
+def test_rounding():
     ffi = FFI()
     ffi.cdef("double sinf(float x);")
     lib = ffi.verify('#include <math.h>', libraries=lib_m)
     res = lib.sinf(1.23)
     assert res != math.sin(1.23)     # not exact, because of double->float
     assert abs(res - math.sin(1.23)) < 1E-5
 
-def test_rounding_2():
-    ffi = FFI()
-    ffi.cdef("double sin(float x);")
-    lib = ffi.verify('#include <math.h>', libraries=lib_m)
-    res = lib.sin(1.23)
-    assert res != math.sin(1.23)     # not exact, because of double->float
-    assert abs(res - math.sin(1.23)) < 1E-5
-
```

Comment:
This test is the same as the one above it

Reply:
Er, no?  One is testing the 'sin' and the other the 'sinf' function.

Reply:
Ok, restoring